src/Sarif.Multitool.Library/Rules/SarifValidationSkimmerBase.cs

```diff
@@ -56,7 +56,7 @@ protected void LogResult(string jPointer, string formatId, params string[] args)
 
             Context.Logger.Log(this,
                 RuleUtilities.BuildResult(
-                    DefaultConfiguration.Level,
+                    level: null,
```

Comment:
>level: null, [](http://example.com/codeflow?start=20&length=12)

this change is because we were passing the DefaultConfiguration.level and write duplicated in the Result.Level,
don't need to since we can leave out the Result.Level json node now.